R/declare_estimator.R

```diff
@@ -264,29 +266,30 @@ model_handler <-
     
     # following copied from dplyr:::as_inlined_function and dplyr:::as_fun_list
     
-    if(rlang::is_formula(post_estimation)) {
+    if(is_formula(post_estimation)) {
```

Comment:
I'd recommend pulling the formula/function/character logic out into a helper function.